soco/core.py

```diff
@@ -479,6 +480,36 @@ def ramp_to_volume(self, volume, ramp_type='SLEEP_TIMER_RAMP_TYPE'):
         ])
         return int(response['RampTime'])
 
+    def set_relative_volume(self, relative_volume):
+        """Adjust the volume up or down by a relative amount.
+
+        If the adjustment causes the volume to overshoot the maximum value
+        of 100, the volume will be set to 100. If the adjustment causes the
+        volume to undershoot the minimum value of 0, the volume will be set
+        to 0.
+
+        This method is an alternative to using addition and subtraction
+        assignment operators (+=, -=) on the `volume` property of a `SoCo`
+        instance. These operators perform the same function as
+        `set_relative_volume()` but require two network calls per operation
+        instead of one.
+
+        Args:
+            relative_volume (int): The relative volume adjustment. Can be
+                positive or negative.
+
+        Returns:
+            int: The new volume setting.
+        """
+        relative_volume = int(relative_volume)
```

Comment:
Ok. So although this is hardly the standard for all code in soco (or even in Python in general), we might as well look to improve a little on the existing standard. In the Sphinx documentation-from-docstrings mechanics there is also the support for documenting exceptions that might be raised. I think this is a prime candidate, since the int call will fail with a `ValueError` if whatever you pass in cannot be converted to an int. So, as a last change I will suggest adding to the docstring, between the Args and the Returns, two lines like this:

```python
Raises:
    ValueError: If `relative_volume` cannot be cast as an integer
```